subsys/bluetooth/controller/ll_sw/nordic/lll/lll_df_types.h

```diff
@@ -75,7 +75,7 @@ struct lll_df_adv_cfg {
  * To mitigate the limited accuracy and losing information about saturated IQ samples a 0x80 value
  * is selected to serve the purpose.
  */
-#define IQ_SAMPLE_STATURATED_16_BIT 0x8000
+#define IQ_SAMPLE_STATURATED_16_BIT ((int16_t)0x8000)
 #define IQ_SAMPLE_STATURATED_8_BIT 0x80
```

Comment:
Could you add `((int8_6)0x80)` to this line? That will make sure there is no similar issue if one uses the second macro in a context where integer promotions are applied.

Reply:
Actually did that first, but removed, have now added it again and hopefully fixed the signed off message